man/man1/fi_strerror.1

```diff
@@ -0,0 +1,25 @@
+.TH fi_strerror 1 "2016\-06\-30" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
+.SH NAME
+.PP
+fi_strerror - display libfabric error strings
+.SH SYNOPSIS
+.IP
+.nf
+\f[C]
+fi_strerror\ FI_ERROR_CODE
+\f[]
+.fi
+.SH DESCRIPTION
+.PP
+Display the error string for the given numeric \f[C]FI_ERROR_CODE\f[].
+\f[C]FI_ERROR_CODE\f[] may be a hexadecimal, octal, or decimal constant.
+For convenience, the absolute value of \f[C]FI_ERROR_CODE\f[] will be
+used, since libfabric APIs typically return negative error codes, but
+\f[C]fi_strerror\f[](3) accepts only positive error codes.
```

Comment:
Yes - that's clearer.  Maybe even swap the statement around.  E.g.

Note that although the fi_strerror function only accepts positive error values, for convenience callers may pass in positive or negative error values...

The 'but' clause (snicker, snicker) through me off because it focused my attention there (snicker, snicker), rather than the first part of the sentence.